### PR TITLE
chore: Add CONTRIBUTING; Fix README

### PR DESCRIPTION
**Pull request by SpyTec**
Updates README with more info. As well as a new CONTRIBUTING file

As it's written currently it depends on work done on #48 and #55

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Contributing to convert2rhel
+
+## Coding guidelines
+1. All python code must be python 2.7 compatible
```
Comment (bocekm) [inline CONTRIBUTING.md:4]: Actually, because of conversions to RHEL 6, it must be also 2.6 compatible.

**Comment by SpyTec** [comment]
I didn't expect a README change to affect anything. I'll update it to point to MD file if that works

**Comment by bocekm** [comment]
Aha, two more lines to change:
https://github.com/oamg/convert2rhel/blob/master/packaging/centos_ol/convert2rhel.spec#L100
https://github.com/oamg/convert2rhel/blob/master/packaging/epel/convert2rhel.spec#L89

**Comment by SpyTec** [comment]
> Aha, two more lines to change:
> [/packaging/centos_ol/convert2rhel.spec@`master`#L100](https://github.com/oamg/convert2rhel/blob/master/packaging/centos_ol/convert2rhel.spec#L100) [](/oamg/convert2rhel/blob/HEAD@{2020-05-26T10:51:31Z}/packaging/centos_ol/convert2rhel.spec#L100)
> [/packaging/epel/convert2rhel.spec@`master`#L89](https://github.com/oamg/convert2rhel/blob/master/packaging/epel/convert2rhel.spec#L89) [](/oamg/convert2rhel/blob/HEAD@{2020-05-26T10:51:31Z}/packaging/epel/convert2rhel.spec#L89)

Saw that and commited a change. Somehow it didn't reflect in the build though..

**Pull request merged**